package.json

```diff
@@ -21,5 +21,9 @@
   "files": [
     "github-btn.html",
     "LICENSE.md"
-  ]
+  ],
+  "devDependencies": {
+    "gulp": "^3.8.11",
+    "gulp-html-minifier": "^0.1.6",
```

Comment:
Oh, also, you seem to use a fork of the original repo... https://github.com/jonschlinkert/gulp-htmlmin